examples/qasm.ipynb

```diff
@@ -0,0 +1,394 @@
+{
```

Comment:
<blockquote>To read it, we use the <code>read_qasm</code> function with the arguments being the file path,</blockquote><p>Maybe its better to use "To import it"? The term "read" might be confused with read a string from file<code>qasm_file.read()</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qutip/qutip-notebooks/pull/111/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>